x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_table_panel.config.ts

```diff
@@ -18,15 +18,27 @@ export const vulnerabilityDashboardTableContent = {
     title: i18n.translate('xpack.csp.cnvmDashboardTable.panel.topVulnerableResources.title', {
       defaultMessage: 'Top 10 vulnerable resources',
     }),
+    viewAllLinkText: i18n.translate(
+      'xpack.csp.vulnerabilityTable.panel.allVulnerableResourcesLinkText',
+      {
+        defaultMessage: 'View all vulnerable resources',
+      }
+    ),
   },
   [DASHBOARD_TABLE_TYPES.TOP_PATCH_VULNERABILITIES]: {
     title: i18n.translate('xpack.csp.cnvmDashboardTable.panel.topPatchVulnerabilities.title', {
       defaultMessage: 'Top 10 patchable vulnerabilities',
     }),
+    viewAllLinkText: i18n.translate('xpack.csp.vulnerabilityTable.panel.buttonText', {
+      defaultMessage: 'View all vulnerabilities',
+    }),
```

Comment:
I saw that you changed the nav hook to navigate to table which filters with fixed versions, which is great. i would also suggest to change the text to `View all patchable vulnerabilities`, similar to what you did with `View all vulnerable resources`.

Reply:
updated